resources/web/wwi/viewer.js

```diff
@@ -627,7 +632,9 @@ function populateViewDiv(mdContent) {
   // markdown to html
   window.mermaidGraphCounter = 0;
   window.mermaidGraphs = {};
-  const converter = new showdown.Converter({tables: 'True', extensions: ['wbTabComponent', 'wbRobotComponent', 'wbSpoiler', 'wbChart', 'wbVariables', 'wbAPI', 'wbFigure', 'wbAnchors', 'wbIllustratedSection', 'youtube']});
+  const converter = new showdown.Converter({tables: 'True',
+    extensions: ['wbTabComponent', 'wbRobotComponent', 'wbSpoiler',
+      'wbChart', 'wbVariables', 'wbAPI', 'wbFigure', 'wbAnchors', 'wbIllustratedSection', 'youtube']});
```

Comment:
The formatting is not very readable, maybe it could be extended to something like
```suggestion
  const converter = new showdown.Converter({tables: 'True',
    extensions: [
      'wbTabComponent', 'wbRobotComponent', 'wbSpoiler', 'wbChart', 'wbVariables', 'wbAPI', 'wbFigure', 'wbAnchors',
      'wbIllustratedSection', 'youtube'
    ]});
```